Source/RapyutaSimulationPlugins/Public/Robots/RRRobotStructs.h

```diff
@@ -1374,8 +1464,11 @@ struct RAPYUTASIMULATIONPLUGINS_API FRRRobotModelInfo : public TSharedFromThis<F
         }
     }
 };
-// This class will co-parent with [IFastXmlCallback], which is not a USTRUCT,
-// thus it could not be also either.
+
+/**
+ * @brief  This class will co-parent with [IFastXmlCallback], which is not a USTRUCT,  thus it could not be also either.
```

Comment:
`thus it could not be also either`
Is there a word missing?

Reply:
@Tadinu ?

Reply:
Yeap we could remove that redundant part starting from `thus`